include/openPMD/benchmark/mpi/MPIBenchmark.hpp

```diff
@@ -274,7 +270,7 @@ namespace openPMD
         }
         for( Datatype dt: datatypes )
         {
-            switchType(
+            switchType< BenchmarkExecution< Clock > >(
                 dt,
                 exec,
```

Comment:
Bit confused here, do we still need the `exec` parameter passed then?

Reply:
Ah, looks like it below.

Reply:
Yup, this one is for proving the comment above ;) "stateful information can be passed via the variadic parameters of the signature"